youtube_dl/extractor/tvnoe.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+import re
+from youtube_dl.compat import compat_urlparse
+from .common import InfoExtractor
+from ..utils import clean_html, get_element_by_class
+
+SRC_PATTERN = r'<script.+jwplayer.*sources:\s*(\[[^]]+\])'
+
+
+class TVNoeIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?tvnoe\.cz/video/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'http://www.tvnoe.cz/video/10362',
+        'md5': 'aee983f279aab96ec45ab6e2abb3c2ca',
+        'info_dict': {
+            'id': '10362',
+            'ext': 'mp4',
+            'series': 'Noční univerzita',
+            'title': 'prof. Tomáš Halík, Th.D. - ' +
+                     'Návrat náboženství a střet civilizací',
+            'description': 'md5:f337bae384e1a531a52c55ebc50fff41',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        iframe_url = self._search_regex(r'<iframe[^>]+src="([^"]+)"',
+                                        webpage, 'iframe src attribute')
+
+        ifs_page = self._download_webpage(iframe_url, video_id)
+        ifs_sources = self._search_regex(SRC_PATTERN, ifs_page, 'sources')
+        ifs_sources = json.loads(re.sub(r'(file|label):', r'"\1":',
+                                        ifs_sources))
+
+        assert len(ifs_sources) > 0
+
+        formats = [{'url': compat_urlparse.urljoin(iframe_url, source['file']),
+                    'format_id': source['label']}
+                   for source in ifs_sources]
```

Comment:
This website uses JWPlayer. Use ```JWPlatformBaseIE._extract_jwplayer_data``` instead. Usually it's not a good practice to re-invent the wheel.